WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -223,19 +223,20 @@
         [
           t('.events'),
           true,
-          @competition.events.select(&:official?),
-          Event.all_official,
+          @competition.event_picker_events,
```

Comment:
Ah sure, it's funny that I've made the `associated_events_picker` and haven't thought about this. Clever!